### ISSUE TITLE
Icon visibility at minimum zoom

### ISSUE THREAD

**Issue by amazansky** [todd-lm/nycbikemap#1]
Awesome project! One idea I had—when the user is zoomed all the way out on the map, I don't think they should see icons. Because the icons stay the same size no matter how zoomed the map is, the map can get cluttered when the user zooms out.

At minimum zoom (or at some arbitrary threshold) the icons should disappear so they don't take up a lot of space; better yet, let the user set a custom threshold in the side menu!

**Comment by todd-lm** [comment]
Yeah, a lot of the new icons are starting to make the map pretty crowded. I'll definitely make some of them not visible from a certain zoom threshold and add an option to filter out what you don't want to see.
